### ISSUE TITLE
FR: Option to hide errors when pint/prettier aren't installed

### ISSUE THREAD

**Issue by jacksleight** [robsontenorio/voltage#4]
### Content

Voltage works great with Laravel projects, however the code formatter throws errors in vanilla PHP projects where pint and prettier aren't installed:

![Screenshot 2024-12-12 at 16 27 06](https://github.com/user-attachments/assets/006784b5-8a52-4ab3-9008-979cc3e7ce97)

I know I could install those packages but I have quite a lot of older projects (including non-Laravel ones) where code formatting has never been enforced and I don't really want to start doing it now (at least not all at once).

But, I do want the syntax highlighting that Voltage provides, so don't want to disable the extension entirely in those projects.

Is there a way these errors could be suppressed, or just not pop up? They get quite annoying when you have `formatOnSave` enabled (which I want for all the newer projects).

I also get this error in these vanilla PHP projects, but I'm not exactly sure what it relates to:

![Screenshot 2024-12-12 at 16 33 54](https://github.com/user-attachments/assets/cbb915bc-60a0-400a-bb34-001c7cf222da)

**Comment by robsontenorio** [comment]
If you don't want to "rely" on Voltage formatter, just **remove** this from your settings.
Otherwise the plugin will try to use Pint and Prettier.

```
"[php]": {
    "editor.defaultFormatter": "robsontenorio.voltage"
}
```

**Comment by jacksleight** [comment]
Well, I want to use it 90% of the time, I'd just like to hide those error popups in projects where those dependencies aren't relevant.

I guess I can just disable Voltage in those projects.

**Comment by robsontenorio** [comment]
I think it is the best approach, to disable the plugin.

Once we add a new option to disabled formatting it will end disabling for all workspaces. 
So disable the plugin is the fast option when you switch projects.


But, I will add a option to ignore errors and don't show up them, even with the plugin and formatter enabled.

**Comment by robsontenorio** [comment]
Released 1.3.0
